spec/turns_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/turn'
+require './lib/card'
+
+RSpec.describe Turn do
+
+    before :each do
+        @card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+        @turn = Turn.new("Saturn", @card)
+    end
+    
+    it 'exists' do
+
+        expect(@turn).to be_instance_of(Turn)
+    end
+
+    it 'returngs a card' do
+
+        expect(@turn.card).to eq(@card)
+    end
+
+    it 'returns a guess' do
+
+        expect(@turn.guess).to eq("Saturn")
+    end
+
+    it 'is_guess_correct?' do
+
```

Comment:
i think it would benefit you to not only check for when a guess is incorrect but also when a guess if correct as well to make sure the methods `correct?` and `feedback` are both doing what is expected in both scenarios.